mithril.js

```diff
@@ -1706,16 +1706,16 @@
 		var currentTarget = e.currentTarget || e.srcElement
 		var args
 
+		while (currentTarget && !/^a$/i.test(currentTarget.nodeName)) {
```

Comment:
While we're at it, wouldn't it be easier (computstionally) to just test for `currentTarget.nodeName != 'a' && currentTarget.nodeName != 'A'`?